src/Razor/src/Microsoft.AspNetCore.Razor.OmniSharpPlugin.StrongNamed/OmniSharpProjectSnapshotManagerDispatcher.cs

```diff
@@ -13,7 +11,9 @@ namespace Microsoft.AspNetCore.Razor.OmniSharpPlugin;
 
 public abstract class OmniSharpProjectSnapshotManagerDispatcher
 {
+#pragma warning disable CS8618 // Non-nullable field must contain a non-null value when exiting constructor. Consider declaring as nullable.
     internal ProjectSnapshotManagerDispatcher InternalDispatcher { get; private protected set; }
```

Comment:
I think this sort of thing warrants discussion, which warrants nullability being in a separate PR